moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -33,13 +33,13 @@
 # Author: Ioan Sucan, Felix Messmer
 
 import rospy
-import conversions
+from . import conversions
```

Comment:
Is this to enforce loading from the current folder first? Isn't that done automatically?

Reply:
In Python 3, implicit relative imports [are gone](https://www.python.org/dev/peps/pep-0404/#imports). Python 3.7 threw an ImportError before the change.